command-not-found

```diff
@@ -94,23 +95,18 @@ def main():
                 print("No candidates found\n\n")
     else:
         print(f"Searching the Portage File List for packages installing {cmd}...\n")
-        found = False
-        for path in paths:
-            x = path + "/" + cmd
-            try:
-                ret, out = e_file.e_file.run([x])
-            except SystemExit:
-                # If something is not working just quit
-                break
-            else:
-                if ret == 0:
-                    found = True
-                    sys.stdout.write(out)
-
-        if found:
-            print("")
-        else:
-            print("No candidates found\n\n")
+        try:
+            options = {
+                'file': cmd
+            }
+            ret, out = pfl.e_file.run(options)
+            if ret == 0:
+                found = True
+                sys.stdout.write(out)
+                print("")
+        except SystemExit:
+            # If something is not working just quit
+            print("No candidates found\n\n")    
```

Comment:
But this is going to match every package that installs a file named `cmd` no matter where it is installed. For this use case we are only interested in packages that install `cmd` into `$PATH`. That is why the original code searches specifically for whole paths not just the filename.

This would also introduce a difference in behaviour between the direct importing method and the compatibility fallback method where we just call `/usr/bin/e-file` directly.